src/Sdk/WebApi/WebApi/Results/ResultsHttpClient.cs

```diff
@@ -0,0 +1,156 @@
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using GitHub.Services.Results.Contracts;
+using GitHub.Services.OAuth;
+using System.Net.Http.Formatting;
+using Sdk.WebApi.WebApi;
+using GitHub.Services.Common;
+
+namespace GitHub.Services.Results.Client
+{
+    public class ResultsHttpClient : RawHttpClientBase
+    {
+         public ResultsHttpClient(
+            Uri baseUrl,
+            VssOAuthCredential credentials)
+            : base(baseUrl, credentials)
+        {
+        }
+
+        public ResultsHttpClient(
+            Uri baseUrl,
+            VssOAuthCredential credentials,
+            RawClientHttpRequestSettings settings)
+            : base(baseUrl, credentials, settings)
+        {
+        }
+
+        public ResultsHttpClient(
+            Uri baseUrl,
+            VssOAuthCredential credentials,
+            params DelegatingHandler[] handlers)
+            : base(baseUrl, credentials, handlers)
+        {
+        }
+
+        public ResultsHttpClient(
+            Uri baseUrl,
+            VssOAuthCredential credentials,
+            RawClientHttpRequestSettings settings,
+            params DelegatingHandler[] handlers)
+            : base(baseUrl, credentials, settings, handlers)
+        {
+        }
+
+        public ResultsHttpClient(
+            Uri baseUrl,
+            VssOAuthCredential credentials,
+            string token)
+            : base(baseUrl, credentials)
```

Comment:
why do you need a `token` and `VssOAuthCredential`? what is your token format? JWT/SAS/GitHub PAT?

Reply:
Updated to remove the VssOAuthCredential